lib/editor/components/pattern/PatternStopCard.js

```diff
@@ -522,7 +551,7 @@ class PatternStopContents extends Component<Props, State> {
             </FormGroup>
           </Col>
         </Row>
-        {feedSource.flexUIFeaturesEnabled && pickupDropOffRow}
+        {feedSource.flexUIFeaturesEnabled && this._renderPickupDropOffTypes(false)}
```

Comment:
This condition should be inverted or placed in a ternary with the condition below right? Right now this displays nothing if `flexUIFeaturesEnabled` is false: 
<img width="265" alt="image" src="https://user-images.githubusercontent.com/63798641/208182927-09007644-e2c0-412c-8be7-9902798715e5.png">

Reply:
Is `feedSource.flexUIFeaturesEnabled` true in your case?

Reply:
I can change it if that's needed for non-flex feeds.

Reply:
If we test a non-flex feed on the flex branch, it will be false and we'll only see the screenshot above. We'll need it to still look the same as in current PROD: 
<img width="274" alt="image" src="https://user-images.githubusercontent.com/63798641/208194873-49e031b4-2533-41f9-a5f7-db2905ed84d8.png">

Reply:
Okay, I think I addressed this in https://github.com/ibi-group/datatools-ui/pull/889/commits/88d32d5ae870cccaea5a15e71b1b258463b026b8. The only difference is that, for a non-flex route in a flex-enabled feed, the booking rules fields will still appear (and default with blank).